cin_validator/rules/cin2022_23/rule_8750.py

```diff
@@ -0,0 +1,166 @@
+from typing import Mapping
+
+import pandas as pd
+
+from cin_validator.rule_engine import CINTable, RuleContext, rule_definition
+from cin_validator.test_engine import run_rule
+
+# Get tables and columns of interest from the CINTable object defined in rule_engine/__api.py
+
+ChildIdentifiers = CINTable.ChildIdentifiers
+LAchildID = ChildIdentifiers.LAchildID
+PersonBirthDate = ChildIdentifiers.PersonBirthDate
+ExpectedPersonBirthDate = ChildIdentifiers.ExpectedPersonBirthDate
+GenderCurrent = ChildIdentifiers.GenderCurrent
+
+# define characteristics of rule
+@rule_definition(
+    # write the rule code here, in place of 8500
+    code=8750,
+    # replace ChildIdentifiers with the value in the module column of the excel sheet corresponding to this rule .
+    module=CINTable.ChildIdentifiers,
+    # replace the message with the corresponding value for this rule, gotten from the excel sheet.
+    message="Gender must equal 0 for an unborn child",
+    # The column names tend to be the words within the < > signs in the github issue description.
+    affected_fields=[GenderCurrent, ExpectedPersonBirthDate],
```

Comment:
I think this should also include PersonBirthDate?

Reply:
It feels excessive to include and highlight all three fields for this error.  On reflection I think I should probably change it to GenderCurrent and **PersonBirthDate** instead of GenderCurrent and ExpectedPersonBirthDate, on the basis that the primary identifier of "unborn" is lack of DOB.

![image](https://user-images.githubusercontent.com/89449804/199018577-de9d515c-f065-4e4b-b379-da6563a40a1f.png)

Reply:
So, it's to do with which cells are highlighted on the front end and I just want to check with Tambe which method will give the expected behaviour! Obviously, there shouldn't be any instances where someone has an expected birth date and a DoB, but it's more to do with what the front end expects for the outputs.